prog-o-meter.py

```diff
@@ -48,6 +48,7 @@ def __init__(self, _days, _filename, _username):
         self.root = Tk.Tk()
         self.filename = _filename
         self.username = _username
+        self.logname = _logname

```

Comment:
It doesn't look like we use logname for anything inside ProgressGUI, so do we actually need is as an attribute to this class? Or did I overlook something?

Reply:
The logname attribute is used inside the function log_entry: inside this function, on line 179, we pass self.logname to the function update_log_file, which updates the log file with the text entered by the user.

Reply:
Cool, I'm sorry I had overlooked this